jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -67,10 +67,11 @@
     private Boolean _persistent = null;
 
     private final int _send;
+    private final boolean _chunkEOFContent;
```

Comment:
Wait, I misread that... so response.setHeader("Transfer-Encoding", "chunked") in an HTTPServlet would control it. That does sound like a clean solution.